src/FsCheck/Arbitrary.fs

```diff
@@ -555,6 +563,32 @@ module Arb =
                             
             fromGenShrink (Gen.array2DOf generate,shrinkArray2D)
 
+        ///Generates arbitrary strings, which are lists of chars generated by Char.
+        static member String() = 
+            let generator = Gen.frequency [(9, Gen.map (fun (chars:char[]) -> new String(chars)) generate);(1, Gen.constant null)]
+            let shrinker (s:string) = 
+                    match s with
+                    | null -> Seq.empty
+                    | _ -> s.ToCharArray() |> shrink |> Seq.map (fun chars -> new String(chars))
```

Comment:
No, no change besides that. Removing the toList just makes it use the shrinker for array instead.